packages/d/dpp/port/xmake.lua

```diff
@@ -1,14 +1,32 @@
 add_rules("mode.debug", "mode.release")
 
-add_requires("fmt", "nlohmann_json", "libsodium", "libopus", "openssl", "zlib")
+add_requires("fmt", "nlohmann_json", "openssl", "zlib")
+
+option("coro", {default = false})
+
```

Comment:
remove empty line

Reply:
Resolved the issue in [this commit](https://github.com/xmake-io/xmake-repo/pull/4824/commits/371cb43eaf7abc11bbca8b2e5552821e776f5908)